default/stringtables/en.txt

```diff
@@ -2851,10 +2851,16 @@ MAP_PROD_WASTED_TITLE
 Unused Production
 
 # %1% number of production points generated.
-# %2% number of production points currently not used for production.
+# %2% number of production points currently neither used for production nor for stockpile.
+# %3% yield ratio of importing PP into the imperial stockpile (the other part goes to waste)
+# %4% number of production points transferred to the imperial stockpile.
+# %5% number of production points currently neither used for production nor for stockpile.
 MAP_PROD_WASTED_TEXT
 '''Total Production : %1%
-Wasted Production : %2%
+Overproduction : %2%
+Stockpile Import Ratio: %3%
```

Comment:
"Import" unclear

Reply:
Yeah. I know. Will start a poll on the forums if i should go inverse and call it Waste Ratio